zerver/management/commands/reupload_emoji.py

```diff
@@ -0,0 +1,44 @@
+import io
+from typing import Any
+from urllib.parse import urljoin
+
+from django.core.management import CommandError
+from django.core.management.commands import sendtestemail
+
+from zerver.lib.emoji import get_emoji_url
+from zerver.lib.outgoing_http import OutgoingSession
+from zerver.lib.upload import upload_emoji_image
+from zerver.models import RealmEmoji, UserProfile
+
+
+class Command(sendtestemail.Command):
+    def handle(self, *args: Any, **kwargs: str) -> None:
+        session = OutgoingSession(role="reupload_emoji", timeout=3, max_retries=3)
+
+        for realm_emoji in RealmEmoji.objects.order_by("id"):
```

Comment:
Should we perhaps add an option to the command to run up from a specified id? ``--run-from-id 123``. It might be useful if the process fail halfway through, to not re-run it on all the emoji that already got processed.

Reply:
I think that would probably end up being handy, and not hard to add, so I think that'd a good idea.

Reply:
Maybe `--start-from=17`

Reply:
Added